package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "novuhq",
   "private": true,
-  "packageManager": "pnpm@8.9.0",
+  "packageManager": "pnpm@9.1.1",
   "scripts": {
     "bootstrap": "npm run setup:dev",
     "start": "npm run jarvis",
     "preinstall": "npx only-allow pnpm",
     "prepare": "husky install",
-    "publish": "lerna publish from-package",
-    "setup:project": "npx --yes pnpm@8.9.0 i && node scripts/setup-env-files.js && pnpm build",
+    "publish": "nx release version && git add . && git commit -m 'chore: release' && pnp--dry-run",
```

Comment:
what exactly is the `pnp--dry-run` command? It looks strange to have something with 'dry run' in the name for s a script that is called publish

Reply:
This was a bug - fixed!